code/game/objects/items/weaponry.dm

```diff
@@ -659,6 +662,11 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
 			if(istype(target, /mob/living/))
 				var/mob/living/bug = target
 				bug.death(1)
+		if(is_type_in_typecache(target, spider_panic))
+			to_chat(user, "<span class='warning'>You easily land a critical blow on the [target].</span>")
+			if(istype(target, /mob/living/))
+				var/mob/living/bug = target
+				bug.adjustBruteLoss(-80)
```

Comment:
isn't this healing them

and what the hell no
Fun gimmicks are fun, hell I'd agree with 15-20 damage, but why the hell should this do the shotgun's equivalent of damage against spiders?

Reply:
Spiders have over 100 hp is why, a welder deals around 23 iirc do to their armor values.
ill have it do 40